modules/installation-two-node-ingress-lb-configuration.adoc

```diff
@@ -0,0 +1,71 @@
+:_mod-docs-content-type: PROCEDURE
+[id="two-node-ingress-lb-configuration_{context}"]
+= Configuring an Ingress load balancer for a two-node cluster with fencing
+
+You must configure an external Ingress load balancer (LB) before you install a two-node OpenShift cluster with fencing. The Ingress LB forwards external application traffic to the Ingress Controller pods that run on the control plane nodes. Both nodes can actively receive traffic.
+
+.Prerequisites
+
+* You have two control plane nodes with fencing enabled.  
+* You have network connectivity from the load balancer to both control plane nodes.  
+* You created DNS records for `api.<cluster_name>.<base_domain>` and `*.apps.<cluster_name>.<base_domain>`.  
+* You have an external load balancer that supports health checks on endpoints.  
+
+.Procedure
+
+. Configure the load balancer to forward traffic for the following ports:
++
+* **6443**: Kubernetes API server  
```

Comment:
These asterisk issues might create unwanted bold formatting in the customer docs. Consider experimenting with 

`\*`